pulpcore/pulpcore/app/models/repository.py

```diff
@@ -170,15 +166,14 @@ class Exporter(MasterModel):
     Fields:
 
         name (models.CharField): The exporter unique name.
-        last_updated (models.DatetimeField): Timestamp of the last update.
         last_export (models.DatetimeField): When the last successful export occurred.
 
     Relations:
 
     """
     TYPE = 'exporter'
 
-    name = models.CharField(max_length=255, db_index=True, unique=True)
```

Comment:
This should probably be a TextField like all the others.

Reply:
...or maybe **all** the names should be CharField, instead.  TextField is really meant for long blobs of text, CharField for short ones that need to be size-bounded. 

We just need to be consistent.

Reply:
I don't think we know the max length of all strings, so that change would be pretty big. https://docs.pulpproject.org/en/3.0/nightly/contributing/3.0-development/data-modeling.html?highlight=textfield#introduction

Reply:
+1.